File: Diamonds/diamonds.js
var x= prompt("Enter a number of rows: ");

function drawDiamond(x){
    for(let i=1; i<=x; i++){
    var r= 2*i-1;
    var stars= "*".repeat(r);
    var spaces= " ".repeat(x-i);
    console.log(spaces+stars);  
    }

    for(let i=2*x-3; i>=1; i-=2){
        var stars="*".repeat(i);
        var spaces=" ".repeat((2*x-1-i)/ 2);
        console.log(spaces+stars);
    }
}

drawDiamond(x);

File: Diamonds/diamonds.py
x = int(input("Enter a number of rows: "))

def drawDiamond(x):
    for i in range(1, x+1):
        r=2*i-1
        stars="*"*r
        spaces=" "*(x-i)
        print(spaces+stars)

    for i in range(2*x-3, 0, -2):
        stars="*"*i
        spaces=" "*((2*x-1-i)//2)
        print(spaces+stars)
        
drawDiamond(x)
